usr/share/tb-profile-i2p/start-i2p-browser

```diff
@@ -0,0 +1,373 @@
+#!/usr/bin/env bash
```

Comment:
Could you please send a pull request / patch to Tor Project making start-tor-browser script configurable by environment variables so we can abolish this script in future?

reasons:

* code deduplication
* not missing out on upstream changes to start-tor-browser
* less maintenance work (in case of changes to start-tor-browser)
* less code inside Whonix

Reply:
Absolutely. I hate that it has to exist for now. I will get that started.